<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="PhoneBookLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>290</width>
        <height>76</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>PHONEBOOK</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="AddPushButton">
           <property name="font">
            <font>
             <family>Cambria</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ADD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SearchPushButton">
           <property name="font">
            <font>
             <family>Cambria</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>SEARCH</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="RemovePushButton">
           <property name="font">
            <font>
             <family>Cambria</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>REMOVE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ListPushButton">
           <property name="font">
            <font>
             <family>Cambria</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>LIST</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="AddLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>151</width>
        <height>76</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ADD </string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddSubmitPushButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>240</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddMainPushButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>MAIN</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>341</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="AddNameLabel">
           <property name="text">
            <string>NAME</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="AddNameLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="AddNumberLabel">
           <property name="text">
            <string>NUMBER</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="AddNumberLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="AddEmailLabel">
           <property name="text">
            <string>E-MAIL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="AddEmailLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="SearchLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>201</width>
        <height>76</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SEARCH </string>
      </property>
     </widget>
     <widget class="QListView" name="SearchListView">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>271</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SearchPushButton_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>SEARCH</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SearchMainPushButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>120</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>MAIN</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>271</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="SearchNameLabel">
         <property name="text">
          <string>NAME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SearchNameLineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_13">
     <widget class="QLabel" name="RemoveLabel">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>201</width>
        <height>76</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>REMOVE </string>
      </property>
     </widget>
     <widget class="QPushButton" name="RemoveMainPushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>200</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>REMOVE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RemoveMainPushButton_6">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>MAIN</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>200</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="RemoveNameLabel">
         <property name="text">
          <string>NAME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="RemoveLineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_14">
     <widget class="QLabel" name="RemoveLabel_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>LIST </string>
      </property>
     </widget>
     <widget class="QListView" name="AllContactsListView">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>271</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SaveListPushButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>81</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>SAVE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ReloadListPushButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>140</y>
        <width>81</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>RELOAD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ListMainPushButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>180</y>
        <width>81</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>MAIN</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
